Playbooks/AS-Slack-Integration/azuredeploy.json

```diff
@@ -0,0 +1,505 @@
+                                        }
+                                    ],
+                                    "channel": "@variables('ChannelId')"
+                                },
+                                "headers": {
+                                    "Authorization": "Bearer @variables('OAuthToken')",
+                                    "Content-type": "application/json; charset=utf-8"
+                                },
+                                "method": "POST",
+                                "uri": "https://slack.com/api/chat.postMessage"
+                            }
+                        },
+                        "Initialize_Variable_-_Description": {
+                            "runAfter": {
+                                "Condition_-_Check_and_Set_Severity_Icon": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable",
+                            "inputs": {
+                                "variables": [
+                                    {
+                                        "name": "Description",
+                                        "type": "string"
+                                    }
+                                ]
+                            }
+                        },
+                        "Initialize_Variable_-_Entities": {
+                            "runAfter": {
+                                "Condition_-_Check_if_Description_Exists": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable",
+                            "inputs": {
+                                "variables": [
+                                    {
+                                        "name": "Entities",
+                                        "type": "string"
+                                    }
+                                ]
+                            }
+                        },
+                        "Initialize_Variable_-_Severity_Icon": {
+                            "inputs": {
+                                "variables": [
+                                    {
+                                        "name": "Severity Icon",
+                                        "type": "string",
+                                        "value": ""
+                                    }
+                                ]
+                            },
+                            "runAfter": {
+                                "Initialize_variable__-_OAuthToken": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable"
+                        },
+                        "Initialize_variable_-_ChannelId": {

```

Comment:
No need to separately have a variable for Channel Id. Data is already present in parameter and can be used from there.

Reply:
This is resolved